suseconnect/suseconnect.go

```diff
@@ -140,6 +142,9 @@ func connectMain() {
 		output, err := connect.GetProductStatuses("json")
 		exitOnError(err)
 		fmt.Println(output)
+	} else if keepAlive {
+		err := connect.SendKeepAlivePing()
+		exitOnError(err)
```

Comment:
To match https://github.com/SUSE/connect/pull/471/files#diff-e763b73e4093dc04b49fd09cecd63710e0502fa8ccc5b71c9eb64d0723ab0445R87 you would need to add custom error handling block inside `exitOnError()`. There is a `switch` for specific error values where you can add your part for `ErrPingFromUnregistered`, print custom message and exit with wanted code. If you don't do that, `default` case will handle the error, print `SUSEConnect error: <error as string>` and exit with `1`.

Reply:
done